source/Mission.cpp

```diff
@@ -369,9 +369,13 @@ void Mission::Save(DataWriter &out, const string &tag) const
 			out.Write("autosave");
 		if(location == LANDING)
 			out.Write("landing");
-		if(location == ASSISTING)
+		else if(location == SHIPYARD)
```

Comment:
```suggestion
		if(location == SHIPYARD)
```
We actually need to keep both landing and shipyard, dont we?
You dont have to apply this suggestion exactly, but the idea is that you cannot have both landing and assist
But you can have both landing and shipyard

Reply:
The Mission class has a unitary value "location" variable (it's an enum), so it can only have one of those values.  So, no, at the moment, it can't be both landing and shipyard.

Reply:
Right nvm I'm an idiot, shipyard is implicitely landing too